example/src/scenes/CustomBankScreen.js

```diff
@@ -0,0 +1,144 @@
+import React, { PureComponent } from 'react'
+import { View, Text, StyleSheet } from 'react-native'
+import stripe from 'tipsi-stripe'
+import Spoiler from '../components/Spoiler'
+import Button from '../components/Button'
+import testID from '../utils/testID'
+
+export default class CustomBankScreen extends PureComponent {
+  static title = 'Custom Bank'
+
+  state = {
+    loading: false,
+    token: null,
+    params: {
+      routingNumber: '110000000', // required field, 9 digits
+      accountNumber: '000123456789', // required field
+      countryCode: 'us', // required field
+      currency: 'usd', // required field
+      accountType: 'individual', // "company" or "individual";
+      accountHolderName: 'Test holder name',
+      accountHolderType: 'company',
+      fingerprint: 'dghajaryktskat',
+      bankName: 'Sberbank',
+      last4: '0000',
```

Comment:
remove `accountType`, `fingerprint`, `bankName` and `last4` fields, also remove all `required field` comments